boards/arm/nrf52_pca20020/nrf52_pca20020.dts

```diff
@@ -91,6 +91,8 @@
 		gpio-controller;
 		#gpio-cells = <2>;
 		ngpios = <16>;
+		init-out-high = <0xe0e0>;
+		init-out-low = <0x1f1f>;
```

Comment:
It would be good to add a comment here explaining why these pins are configured this way (so that it is not required to look into the git log to find it out).